RiotSwiftUI/Modules/Room/Composer/View/Composer.swift

```diff
@@ -140,11 +140,12 @@ struct Composer: View {
             }
             HStack(alignment: shouldFixRoundCorner ? .top : .center, spacing: 0) {
                 WysiwygComposerView(
-                    focused: $viewModel.focused,
-                    viewModel: wysiwygViewModel
+                    placeholder: viewModel.viewState.placeholder ?? "",
+                    viewModel: wysiwygViewModel,
+                    itemProviderHelper: nil,
+                    keyCommandHandler: handleKeyCommand,
+                    pasteHandler: nil
```

Comment:
I think `itemProviderHelper` and `pasteHandler` should do something otherwise I think that the copy and paste of media wouldn't work.
Eg. try to copy a photo and paste it in the composer.
You could check what EX does to implement this feature.

Reply:
El-iOS handles this differently, so I don't think we need to provide an itemProviderHelper or pasteHandler for the composer.
I fixed an issue in this [commit](https://github.com/vector-im/element-ios/pull/7711/commits/76715486f858c3e3490a7a8957cdf452c5f44fc1) to minimise the composer and hide the keyboard when pasting an image, video or file. Otherwise, the keyboard remained visible, hiding the media preview toolbar button.